message_definitions/v1.0/common.xml

```diff
@@ -1827,7 +1827,7 @@
       </entry>
       <!-- END of user range (31000 to 31999) -->
       <entry value="32000" name="MAV_CMD_CAN_FORWARD" hasLocation="false" isDestination="false">
-        <description>Request forwarding of CAN packets from the given CAN bus to this interface. CAN Frames are sent using CAN_FRAME and CANFD_FRAME messages</description>
+        <description>Request forwarding of CAN packets from the given CAN bus to this component. CAN Frames are sent using CAN_FRAME and CANFD_FRAME messages</description>
```

Comment:
Wrong before *and* after, for ArduPilot.  AP will only send on the channel which this command was received on - but will target the component.

Reply:
Can you suggest an explicit description text fix?